### ISSUE TITLE
关于其他3D数据集效果

### ISSUE THREAD

**Issue by CompliceU** [liuruijin17/CLGo#2]
太棒了，感谢你的开源呢，最近还有一些其他的开源数据集，不知道你有打算在其他数据集上进行训练测试吗

**Comment by liuruijin17** [comment]
Thanks for your interest!
Currently, I am working on other projects, but I also notice there are some great datasets published recently. It would be interesting to test CLGo on them. As long as the results are good enough, I would like to share them!

**Comment by CompliceU** [comment]
@liuruijin17 At present, I tried to use OpenLane data set for testing, but found that the effect was not very ideal. Because the training time was too long, I only extracted part of the data for training. Looking forward to your reply.

**Comment by ChonghaoSima** [comment]
@CompliceU hello there! I wonder if you could share the training/testing code of CLGo on OpenLane, as we're trying to evaluate CLGo on OpenLane as well. Thanks in advance!

**Comment by liuruijin17** [comment]
@CompliceU Yes, current codes are too ancient and do not take the advantage of Pytorch's dataset classs to speed up training. The reason exists since I could only take 1~2 GPU for experiments when I do this work in my college.

However, I am now being an intern and taking more GPUs, so I am working on updating codes. 

Sorry for this inefficient version, please be patient!

**Comment by ChonghaoSima** [comment]
@liuruijin17 Greetings from OpenLane. Do you have a plan to release multi-gpu training code and report a result on OpenLane Dataset?
